evennia/contrib/utils/git/README.md

```diff
@@ -0,0 +1,68 @@
+# In-game Git Integration
```

Comment:
Please rename the folder from `utils/git` to `utils/git_integration/`. Using `git` as a folder name makes it looks like it's, well, something to do with the real git and not a contrib.